### DISCUSSION TITLE
setAttribute in actionRules Row applying the text color not working

### DISCUSSION THREAD

**Discussion by Rajdev2011** [Power-Components/livewire-powergrid#1820] [BUG]
### Livewire PowerGrid version

v6.1.9

### Livewire version

v3.5.18

### Laravel version

10.28

### Which PHP version are you using?

PHP 8.2

### Which Theme are you using?

Tailwind

### Have you published the resource files and customized them? (php artisan vendor:publish)

No

### What is the problem?

I am applying the text-red-color if balance is less than 50 but its not applying the class for the row

### Code snippets

```
  public function actionRules(): array
    {
        return [
            Rule::rows()
                ->when(function ($row) {
                    Log::info('Row with balance :', (array) $row);
                    Log::info('Row with balance condition:'. ($row->balance < 50));
                    return $row->balance < 50;
                })
                ->setAttribute('class', '!text-red-500'),
        ];
    }
```

and  below are logs
    
[2024-12-26 05:10:13] local.INFO: Row with balance : {"reportable_form_id":"FD004R","form_name":"Form D","form_id":"FD004","threshold":125,"received_to_date":250,"used":250,"balance":0} 
[2024-12-26 05:10:13] local.INFO: Row with balance condition:1  
[2024-12-26 05:10:13] local.INFO: Row with balance : {"reportable_form_id":"FE005R","form_name":"Form E","form_id":"FE005","threshold":150,"received_to_date":300,"used":300,"balance":0} 
[2024-12-26 05:10:13] local.INFO: Row with balance condition:1
    

### How do you expect it to work?

total row should be red if balance is less than 50

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by luanfreitasdev** [comment]
Fixed https://github.com/Power-Components/livewire-powergrid/releases/tag/v6.1.10
